### ISSUE TITLE
mergedGrid for 3D with different number of z-planes

### ISSUE THREAD

**Issue by tibuch** [platybrowser/mmb-fiji#1005]
#1004 discusses `mergedGrid` view for 3D data and [297488b](https://github.com/mobie/mobie-viewer-fiji/commit/297488bc68c9dd8149a4e673241c51a4c4d43f6e) provides a fix for 3D sources where all sources have the same number of z-planes.

According to @tischi more work is needed to catch cases where the 3D sources have different number of z-planes.

**Comment by tischi** [comment]
Yes, one option: The user needs to select as the `metadataSource` the one that has the largest number of z-planes and then I have to zero-extend the others.
